integrations/slack/save.go

```diff
@@ -0,0 +1,70 @@
+package slack
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"go.autokitteh.dev/autokitteh/integrations/slack/internal/vars"
+	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.uber.org/zap"
+)
+
+const (
+	headerContentType = "Content-Type"
+	contentTypeForm   = "application/x-www-form-urlencoded"
+)
+
+func (h handler) handleSave(w http.ResponseWriter, r *http.Request) {
+	c, l := sdkintegrations.NewConnectionInit(h.logger, w, r, desc)
+
+	// Check "Content-Type" header.
+	contentType := r.Header.Get(headerContentType)
+	if r.Method == http.MethodPost && !strings.HasPrefix(contentType, contentTypeForm) {
```

Comment:
Actually, I think the web UI uses GET even for non-OAuth connections. Please check with @RonenMars, and anyway try to be as close as possible to GitHub's `pat.go`, or Google Calendar/Forms which also do a 2-step flow.

Reply:
Actually for non-OAuth connections we use POST, so we're good.